packages/clay-css/src/scss/components/_tables.scss

```diff
@@ -28,6 +28,16 @@ th {
 	margin-bottom: $table-margin-bottom;
 	width: 100%;
 
+	&.sticky-head {
```

Comment:
@pat270 @bryceosterhaus do you think it shouldn't have the same nomenclature as the other classes? something like `.table-sticky-head`?

Reply:
Ill defer to @pat270 for whatever he thinks is best

Reply:
Sorry I completely forgot about this. I agree with @matuzalemsteles `table-sticky-head` is consistent with the current naming structure. I'm hesitant to add this because a majority of our tables have `table-responsive` which adds `overflow-x: auto`. `position: sticky` only works when the parent container is `overflow: visible` if the browsers scrollbar is on document. 

My next question is are we ok with sticky header tables overflowing horizontally in smaller screens? In the long run I do think we need to use JS to make the experience worthwhile.